canals/pipeline/descriptions.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+from typing import List, Dict
+import logging
+
+import networkx
+
+from canals.type_utils import _type_name
+from canals.component.sockets import InputSocket, OutputSocket
+
+
+logger = logging.getLogger(__name__)
+
+
+def _find_pipeline_inputs(graph: networkx.MultiDiGraph) -> Dict[str, List[InputSocket]]:
```

Comment:
Let's remove the leading underscore, this method is imported elsewhere shouldn't be marked as private